IntelliTect.Analyzer/IntelliTect.Analyzer.Test/Verifiers/DiagnosticVerifier.cs

```diff
@@ -122,52 +126,35 @@ private static void VerifyDiagnosticResults(IEnumerable<Diagnostic> actualResult
 
                 if (expected.Line == -1 && expected.Column == -1)
                 {
-                    if (actual.Location != Location.None)
-                    {
-                        Assert.IsTrue(false,
-                            string.Format("Expected:\nA project diagnostic with No location\nActual:\n{0}",
-                            FormatDiagnostics(analyzer, actual)));
-                    }
+                    Assert.AreEqual(actual.Location,
+                                    Location.None,
+                                    string.Format($"Expected:\nA project diagnostic with No location\nActual:\n{FormatDiagnostics(analyzer, actual)}"));
```

Comment:
I realize the original code was this way, but I find it interesting that there is a mix of "\r\n" and "\n" in the same file

Reply:
I'll update them to use Environment.NewLine